src/Microsoft.Docs.Template/Models/TableOfContents.cs

```diff
@@ -28,7 +28,7 @@ public class TableOfContentsItem
         public string TocHref { get; set; }
 
         [JsonProperty(PropertyName = "expanded", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public bool Expanded { get; set; }
```

Comment:
why do we need to make it nullable?

Reply:
in v2, toc file with `expanded` property explicitly given as `false`, the `expanded: false` property will be built into the result. in v3, if we use `DefaultValueHandling = DefaultValueHandling.Ignore` attribute, the false property won't be serialized, which leads to a diff.  
Though both `null` and `false` should behave the same.